tools/depmod.c

```diff
@@ -2071,39 +2040,42 @@ static inline const char *mod_get_compressed_path(const struct mod *mod)
 static int output_deps(struct depmod *depmod, FILE *out)
 {
 	size_t i;
+	struct array array;
```

Comment:
same here... which avoids confusion between this array and depmod->modules.array

Reply:
Adjusted